### DISCUSSION TITLE
'Like' search doesn't work

### DISCUSSION THREAD

**Discussion by Pieter0313** [Budibase/budibase#5703] [Help]
I have 2 search filters, both use the 'like' condition so that the shown data only show data that contains the given input.

This is what my data looks like:
<img width="818" alt="image" src="https://user-images.githubusercontent.com/52565351/166697527-c12a4250-f0b3-4a7e-b827-4a30f85bf2ad.png">

This is what the filtering looks like:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/52565351/166697406-9d9999a6-73c7-4089-a2a7-5ba4257e3796.png">

And this is what happens when I try to search:
<img width="817" alt="image" src="https://user-images.githubusercontent.com/52565351/166697681-6848468c-cce3-4777-abd9-ee3ee6b0d11c.png">

As you can see, no data is shown. It only shows if I fill in the exact field values like 'test', but it should work with my configured filters right? I don't know if I did something wrong or if this is broken, because also when I try to use the 'Starts with' condition it does work in the editor, but doesn't in production mode. Would be great if someone could help me out :)

**Comment by gerhardmpl** [comment]
You might check the open GitHub issue #3386. The `Like` option for filters on the internal tables does not work because of a restriction on CouchDB and might even be removed (for internal tables).

**Comment by JayP718** [comment]
Hi all,

It seems like the Like functionality is not working for us either. We are using external sql server tables. Is this a known issue or perhaps we are doing something wrong?
